azure-devops/azext_devops/devops_sdk/client.py

```diff
@@ -102,6 +102,7 @@ def _send(self, http_method, location_id, version, route_values=None,
         if Client._session_header_key in Client._session_data and Client._session_header_key not in headers:
             headers[Client._session_header_key] = Client._session_data[Client._session_header_key]
         response = self._send_request(request=request, headers=headers, content=content, media_type=media_type)
+        response.continuation_token = response.headers.get('X-MS-ContinuationToken')
```

Comment:
Maybe it is included someplace else but when we are making a subsequent call don't we need to send the continuation token in request as well?

Reply:
Checking

Reply:
@aasiddiq 

It is not there actually, the "invoke" command does not have support for continuous token.
We can close this one and I can create a separate "Feature Request" to support "continuous token" in the request header.
And let's track it separately.

Reply:
"Clarification"
Invoke supports continuation token but not in the header, It support as a Query string.